sklearn/ensemble/_base.py

```diff
@@ -128,24 +127,10 @@ def __init__(self, base_estimator, *, n_estimators=10, estimator_params=tuple())
         # self.estimators_ needs to be filled by the derived classes in fit.
 
     def _validate_estimator(self, default=None):
-        """Check the estimator and the n_estimator attribute.
+        """Check the estimator.
```

Comment:
While we're here, minor wording shift for clarity.
```suggestion
        """Check the base estimator.
```

Reply:
Done! :)